src/index.js

```diff
@@ -137,10 +137,10 @@ const unexpectedEnzyme = {
     childExpect.exportAssertion(
       '<ReactWrapper> [not] to have class <string>',
       (expect, reactWrapper, className) => {
-        if (expect.flags.not === reactWrapper.hasClass(className)) {
-          const classes = (reactWrapper.prop('className') || '').split(' ');
-          expect(classes, '[not] to contain', className);
-        }
+        const reactElement = reactWrapper.getDOMNode();
+        const actualClasses = reactElement.className.split(' ');
+
+        expect(actualClasses, '[not] to contain', ...className.split(' '));
```

Comment:
@sunesimonsen do you think we should do this? I'm unsure about the behaviour of `to contain <array>` which is truthy for just a single item being contained. I would expect this to be true only for all the asserted items.

Reply:
I really hope to contain only succeeds if all of the items are present.

Reply:
I really hope to contain only succeeds if all of the items are present.

Reply:
Not according to `unexpected` http://unexpected.js.org/assertions/array-like/to-contain/

> Asserts an array (or array-like object) contains one or more items.

Reply:
That is not what it means, you can ask if one or more items is in the collection. If you ask for 3 then all of them needs to be there.

Reply:
Make a test to be sure.

Reply:
Yeah that works like that